tests/devices_test/dependencies_test.py

```diff
@@ -97,10 +97,6 @@ def test_availability_mdraidplugin(self):
             ActionCreateDevice(self.luks)
         with self.assertRaises(DependencyError):
             ActionDestroyDevice(self.dev)
-        with self.assertRaises(DependencyError):
-            ActionCreateFormat(self.dev)
-        with self.assertRaises(DependencyError):
-            ActionDestroyFormat(self.dev)
```

Comment:
Why remove these if this test doesn't involve btrfs?

Reply:
I've moved this change to a separate commit. This isn't really related to the btrfs changes, but I've noticed that we check the unavailable_dependencies for all actions but these dependencies are defined only for devices (formats use tasks and there are separate checks).
Currently it isn't possible to create (or remove) format on a device we don't support -- e.g. it isn't possible to format (opened) luks device without libblockdev crypto plugin. But the plugin isn't needed to run mkfs on a block device so we should allow it.